cmd/deploy.go

```diff
@@ -170,10 +171,31 @@ func setFlags(conf *clab.Conf) {
 	}
 }
 
-func createHostsFile(containers []types.Container, dirPath string, bridgeName string) error {
+func createHostsFile(containers []types.Container, bridgeName string) error {
 	if bridgeName == "" {
 		return fmt.Errorf("missing bridge name")
 	}
+	data := hostsEntries(containers, bridgeName)
+	if len(data) == 0 {
+		return nil
+	}
+	f, err := os.OpenFile("/etc/hosts", os.O_APPEND|os.O_WRONLY, os.ModeAppend)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	_, err = f.WriteString("\n")
+	if err != nil {
+		return err
+	}
+	_, err = f.Write(data)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func hostsEntries(containers []types.Container, bridgeName string) []byte {
```

Comment:
how about to put a comment to this func, smth like:

hostEntries builds an /etc/hosts compliant text blob (returned as a byte slice) by adding containers IPv4/v6 addresses and mapping them to container names.